templates/package/content/container.tmpl

```diff
@@ -14,6 +14,10 @@
 					<div class="markup"><pre class="code-block"><code>docker pull {{.RegistryHost}}/{{.PackageDescriptor.Owner.LowerName}}/{{.PackageDescriptor.Package.LowerName}}{{$separator}}{{.PackageDescriptor.Version.LowerVersion}}</code></pre></div>
 				{{end}}
 			</div>
+			<div class="field">
+				<label>{{svg "octicon-code"}} {{.locale.Tr "packages.container.digest"}}</label>
+				<div class="markup"><pre class="code-block"><code>{{range .PackageDescriptor.Files}}{{if eq .File.LowerName "manifest.json"}}{{.Properties.GetByName "container.digest"}}{{end}}{{end}}</code></pre></div>
```

Comment:
```suggestion
				<label for="container-image-digest">{{svg "octicon-code"}} {{.locale.Tr "packages.container.digest"}}</label>
				<div class="markup" id="container-image-digest"><pre class="code-block"><code>{{range .PackageDescriptor.Files}}{{if eq .File.LowerName "manifest.json"}}{{.Properties.GetByName "container.digest"}}{{end}}{{end}}</code></pre></div>
```
Should hopefully be more accessible while not changing its display.

Reply:
These should be added in another PR because none of the package labels use that.

Reply:
looking forward to it :point_up:  ;)